dipy/denoise/patch2self.py

```diff
@@ -324,9 +683,9 @@ def patch2self(
 
     # Insert the separately denoised arrays into the respective empty arrays
     for vol_idx in range(0, data_dwi.shape[3]):
-        denoised_dwi[..., vol_idx] = _vol_denoise(
-            train_dwi, vol_idx, model, data_dwi.shape, alpha=alpha
-        )
+        denoised_dwi[..., vol_idx] = _fit_denoising_model(
+            train_dwi, vol_idx, model, alpha=alpha, version=version
```

Comment:
`version=1` instead.   it does not make sense to have other version in `patch2self_version1`